scenarios/01_Elensefar.cfg

```diff
@@ -114,7 +116,7 @@
 
     # Hard difficulty allows this side to recruit blood bats, so the gold and income are much higher to match the more expensive units and upkeep.
     [event]
-        {QUANTITY4 name (unused_event) (unused_event) (turn 13) (turn 13)}
```

Comment:
This is enemy reinforcements arriving, so it should happen later for easier difficulties. I can't remember why I chose to have them arrive at dawn, but probably to avoid lessen the chance of it becoming an ambush against a player who had sent troops to the northern castle. Might also have been so that the bats are likely to reach Elensefar itself during the next night.

Reply:
hm, how about `{(unused_event) (unused_event) (turn 19) (turn 13)}` then?

Reply:
> hm, how about `{(unused_event) (unused_event) (turn 19) (turn 13)}` then?

I went with this in 3c11649, and added a comment explaining.